tests/scripts/thread-cert/node.py

```diff
@@ -940,11 +940,11 @@ def set_parent_priority(self, priority):
         self._expect('Done')
 
     def get_partition_id(self):
-        self.send_command('leaderpartitionid')
+        self.send_command('leaderpartitionid preferred')
         return self._expect_result(r'\d+')
 
     def set_partition_id(self, partition_id):
```

Comment:
rename to be `set_preferred_partitiond_id`?

Reply:
updated